css/properties/font-family.json

```diff
@@ -18,7 +18,9 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "partial_implementation" : true,
```

Comment:
```suggestion
              "partial_implementation": true,
```

Reply:
I'm not sure this rather edge case bug should set the whole `font-family` feature to partial support and thus compat linters could potentially warn about using `font-family` in Firefox. I'd only add the note. What do you think?

Reply:
You strike a good point, especially since the feature as a whole works tremendously in all other elements.  Removal of this line it is!